tests/test_sequence.py

```diff
@@ -21,7 +21,7 @@ def setUp(self):
 
```

Comment:
Is it possible to create a text fixture with the sequences having the ranges defined from _schema.sql_? Then we could test for valid ranges, etc.